microcosm/tests/test_annotations.py

```diff
@@ -0,0 +1,100 @@
+"""
+Annotation tests.
+
+"""
+from hamcrest import (
+    assert_that,
+    equal_to,
+    has_entries,
+    is_,
+)
+
+from microcosm.api import create_object_graph, binding, defaults
+from microcosm.annotations import boolean, must_override, required, split
+from microcosm.decorators import get_defaults
+from microcosm.loaders import load_from_dict
+from microcosm.registry import Registry
+
+
+REGISTRY = Registry()
+
+
+@binding("integer", REGISTRY)
+@defaults(
+    value=int,
+)
+def integer(graph):
+    return graph.config.integer.value
+
+
+@binding("boolean", REGISTRY)
+@defaults(
+    value=boolean,
```

Comment:
Naming in this test is a little confusing. Somehow my brain goes weird places with the key called `value` would probably be much easier if it were called `key`.